docs/porting/third-party-deps.md

```diff
@@ -1,6 +1,6 @@
 # Analyzing your 3rd Party Dependencies
 
-The first step in the porting process is to understand your third party dependencies, figure out which of them (if any), don't run on .NET Core, and develop a contingency plan for those which don't run on .NET Core.
+The first step in the porting process is to understand your third party dependencies.  You need to figure out which of them, if any, don't yet run on .NET Core, and develop a contingency plan for those which don't run on .NET Core.
```

Comment:
nit: third party should have a hyphen. Also, it would be better to use the work instead of 3rd in the title